pkg/management/authorization.go

```diff
@@ -68,6 +68,9 @@ var (
 	MGMTDeleteGroup                         = newAction("MGMT_DeleteGroup", security.ScopeGroup)
 	MGMTGetAuthorizations                   = newAction("MGMT_GetAuthorizations", security.ScopeGroup)
 	MGMTUpdateAuthorizations                = newAction("MGMT_UpdateAuthorizations", security.ScopeGroup)
+	MGMTPutAuthorization                    = newAction("MGMT_PutAuthorization", security.ScopeGroup)
```

Comment:
MGMT_AddAuthorization

Reply:
Corrected in commit e65d1abb21021efc38ddf4adf3ef93c6273bd283